ToyBox/ReadMe.md

```diff
@@ -28,6 +28,9 @@
 * **Crusade Editor**
   * Reworked UI to match design of other parts of the mod
   * Fixed bug where it was displaying the Exp required for the current rank rather than the next rank
+* (***ShadowRanger***) Bugfix for taking 10 out of combat: corrected missing IsInCombat flag
+* (***ShadowRanger***) Added a vescavors-begone option, similar to spiders-begone, to replace all vescavor models 
```

Comment:
If it is convenient, please try to do separate pull requests for separate patches but this is awesome, thanks for the hard work.

Reply:
Ah; might just be me messing up the git forking / merging.  I believe the PR with the bugfix for taking 10 got pulled in yesterday.